examples/interactive_play.py

```diff
@@ -252,6 +262,42 @@ def get_input_vel_ctlr(
             logger.info("[play.py]: Snapping")
             magic_grasp = 1
 
+    if use_joint_control:
+        # Add random noise to human arms but keep global transform
+        (
+            new_joints,
```

Comment:
I think `new_joints` is a bad variable name. Should we call it `joint_trans` to match `root_trans` ?

Reply:
Should be there now